store/sqlstore/container.go

```diff
@@ -177,10 +180,10 @@ const (
 									  signed_pre_key, signed_pre_key_id, signed_pre_key_sig,
 									  adv_key, adv_details, adv_account_sig, adv_device_sig,
 									  platform, business_name, push_name)
-		VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14)
-		ON CONFLICT (jid) DO UPDATE SET platform=$12, business_name=$13, push_name=$14
+		VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
+		ON CONFLICT (jid) DO UPDATE SET platform=?, business_name=?, push_name=?
```

Comment:
Apparently, MySQL (MariaDB) does not do positional placeholders. We must use some repetition. :(